### ISSUE TITLE
so is the idea of SCHEDULER just an easy way to create a job in ruby that runs every whatever interval?

### ISSUE THREAD

**Issue by letsgoduke** [skeletalbassman/Epic-Tracker#1]

**Comment by sulimmesh** [comment]
Yeah essentially. As long as the scheduler persists in memory the schedule will continue. It's also pretty intelligent about intervals; '10m' becomes 10 minutes, '1hr' becomes 1 hour, etc. You can find more about the rufus-scheduler here: https://github.com/jmettraux/rufus-scheduler
